tests/test_config/test_config_tags.py

```diff
@@ -0,0 +1,206 @@
+"""Unit tests specific to the tag-related ConfigService APIs.
+
+ These APIs include:
+   list_tags_for_resource
+   tag_resource
+   untag_resource
+
+"""
+import boto3
+from botocore.exceptions import ClientError
+from botocore.exceptions import ParamValidationError
+import pytest
+
+from moto.config import mock_config
+from moto.config.models import MAX_TAGS_IN_ARG
+from moto.config.models import random_string
+from moto.core import ACCOUNT_ID
+
+TEST_REGION = "us-east-1"
```

Comment:
This isn't used anywhere

Reply:
It is ... as an argument to put_aggregation_authorization().  In `test_config.py` that argument is hard coded.  Here I made it a define.  See line 111.